Dictionaries/task1.py

```diff
@@ -0,0 +1,12 @@
+roles = {'admin': ['Ivan', 'Petro'],
+         'maintainer': ['Dmytro'],
+         'manager': ['Oleksandr'],
+         'developer': ['Maria', 'Sofia']}
+
+name = input('Name: ')
+role = [key for key, value in roles.items() if name in value]
+
+if role:
+    print(f'Hello, {role[0]}')
+else:
+    print(f'Hello, Guest')
```

Comment:
Good variant, but can be other
![image](https://github.com/m0rg1nal/courses/assets/51053483/19500e00-2c34-4e59-82cd-2f908506a52d)